include/podio/RNTupleWriter.h

```diff
@@ -1,25 +1,34 @@
 #ifndef PODIO_RNTUPLEWRITER_H
 #define PODIO_RNTUPLEWRITER_H
 
-#include "podio/Frame.h"
-#include "podio/GenericParameters.h"
+#include "TFile.h"
```

Comment:
Would move this to the other ROOT includes (might be solved by the addition to clang-format above).

Reply:
Done, this has to be done manually